Merge/Dependencies.hs

```diff
@@ -383,6 +383,7 @@ staticTranslateExtraLib lib = lookup lib m
       , ("SDL2_mixer", any_c_p "media-libs" "sdl2-mixer")
       , ("blas", any_c_p "virtual" "blas")
       , ("lapack", any_c_p "virtual" "lapack")
+      , ("ghc", any_c_p "dev-lang" "ghc")
```

Comment:
Why do you need it? `extra-libraries` are libraries used directly in linking, like `-lghc`. `dev-lang/ghc` does not provide such libraries. It's at best `HSghc-8.10.4-ghc8.10.4`.